pkg/snowflake/view.go

```diff
@@ -21,23 +21,12 @@ type ViewBuilder struct {
 
 // QualifiedName prepends the db and schema if set and escapes everything nicely
 func (vb *ViewBuilder) QualifiedName() string {
-	var n strings.Builder
-
-	if vb.db != "" && vb.schema != "" {
-		n.WriteString(fmt.Sprintf(`"%v"."%v".`, vb.db, vb.schema))
-	}
-
-	if vb.db != "" && vb.schema == "" {
-		n.WriteString(fmt.Sprintf(`"%v"..`, vb.db))
+	// should not happen since this is enforced by view resource schema
+	if vb.db == "" || vb.schema == "" {
+		panic("Views must specify a database and a schema") //nolint
```

Comment:
this should be an error, not panic

Reply:
I'll switch over. Since it is unreachable code through normal TF operations I mostly wanted this assertion for anyone making code changes in the future instead.